lib/src/realm_class.dart

```diff
@@ -33,6 +34,7 @@ import 'scheduler.dart';
 import 'subscription.dart';
 import 'session.dart';
 
+export 'package:cancellation_token/cancellation_token.dart' show CancellationToken, CancelledException, CancellableCompleter, CancellableFuture;
```

Comment:
```suggestion
export 'package:cancellation_token/cancellation_token.dart' show CancellationToken, CancelledException;
```

Do we need to expose those extra types?

Reply:
@nielsenko , @blagoev What do you think? Shall we expose `CancellableCompleter` and `CancellableFuture` or not?

Reply:
Personally I don't think we should export classes from other packages. I think they should pull in the package themselves if they wan't to use it.